dace/sdfg/propagation.py

```diff
@@ -546,16 +553,346 @@ def propagate(self, array, expressions, node_range):
+                            # and accumulate the total volume between them.
+                            memlets = []
+                            for edge in edges:
+                                inside_memlet = edge.data
+                                memlets.append(inside_memlet)
+
+                                if memlet is None:
+                                    # Use the first encountered memlet as a
+                                    # 'border' memlet and accumulate the sum
+                                    # on it.
+                                    memlet = copy.deepcopy(inside_memlet)
+                                    memlet.dynamic = False
+                                    memlet.volume = 0
+                                    memlet.subset = None
+                                    memlet.other_subset = None
+                                    memlet._is_data_src = True
+                                    border_memlets[direction][
+                                        node.label
+                                    ] = memlet
+
+                                if memlet.dynamic and memlet.volume == 0:
+                                    # Dynamic unbounded - this won't change.
+                                    continue
+                                elif ((inside_memlet.dynamic and
+                                       inside_memlet.volume == 0) or
+                                      (state.dynamic_executions and
+                                       state.executions == 0)):
+                                    # At least one dynamic unbounded memlet lets
+                                    # the sum be dynamic unbounded.
+                                    memlet.dynamic = True
+                                    memlet.volume = 0
+                                else:
+                                    memlet.volume += (
+                                        inside_memlet.volume * state.executions
+                                    )
+                                    memlet.dynamic = (
+                                        memlet.dynamic or
+                                        inside_memlet.dynamic or
+                                        state.dynamic_executions
+                                    )
+
+                            # Given all of this access nodes' memlets, propagate
+                            # the subset according to the state's variable
+                            # ranges.
+                            if len(memlets) > 0:
+                                params = []
+                                ranges = []
+                                for symbol in state.ranges:
+                                    params.append(symbol)
+                                    ranges.append(state.ranges[symbol][0])
+
+                                if len(params) == 0 or len(ranges) == 0:
+                                    params = ['dummy']
```

Comment:
```suggestion
                                    params = ['__dace_dummy']
```

Reply:
some people might use a dummy variable name!

Reply:
Addressed in 7a3b27b